dockerspawner/dockerspawner.py

```diff
@@ -363,7 +362,9 @@ def escaped_name(self):
 
     @property
     def container_name(self):
-        return "{}-{}".format(self.container_prefix, self.escaped_name)
+        escaped_container_image = self.image.replace("/", "_")
+        d = {'username' : self.escaped_name, 'imagename' : escaped_container_image}
```

Comment:
include container_prefix here, and probably include, `server_name = getattr(self, 'name', '')` in the namespace for the new named servers feature in 0.8.